<?xml version="1.0"?>
<launch>
	<!-- Parameters to pass to the ROS node -->
	<arg name="move_group" default="manipulator"/>
  
	<group ns="iiwa">
		<node pkg="iiwa_grasp" name="Pipe_Line"  type="pipe_line.py" output="screen"/>
		<node pkg="iiwa_grasp" name="Object_Detection"  type="object_detection.py" output="screen"/>
  		<node pkg="iiwa_grasp" name="Pose_Transform"  type="pose_transform.py" output="screen"/>
		<node pkg="iiwa_grasp" name="Move_Robot"  type="move_robot.py" output="screen"/>
		<param name="planning_frame" value="/world"/>
		<param name="move_group" value="$(arg move_group)"/>
	</group>
  
</launch>
